luxonis_train/nodes/backbones/ghostfacenet.py

```diff
@@ -0,0 +1,511 @@
+# Original source: https://github.com/Hazqeel09/ellzaf_ml/blob/main/ellzaf_ml/models/ghostfacenetsv2.py
+
+
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from luxonis_train.nodes.base_node import BaseNode
+
+
+def _make_divisible(v, divisor, min_value=None):
```

Comment:
Check if some of these util functions and blocks are already implemented and can be reused (e.g. ConvBNAct can be replaced with [ConvModule](https://github.com/luxonis/luxonis-train/blob/8be019f7c6924dd902532b8156c09c917eb435b0/luxonis_train/nodes/blocks/blocks.py#L84), then [SqueezeExciteBlock](https://github.com/luxonis/luxonis-train/blob/8be019f7c6924dd902532b8156c09c917eb435b0/luxonis_train/nodes/blocks/blocks.py#L199C7-L199C25) could be used, etc.)